### DISCUSSION TITLE
想把企业微信的客服消息推送到服务器上来，但openapi请求为啥就不通过呢？

### DISCUSSION THREAD

**Discussion by msmax** [w7corp/easywechat#2618] [Q&A]
<img width="623" alt="image" src="https://user-images.githubusercontent.com/15795938/201481374-c0fe8d37-6526-4810-bbca-da3821513a49.png">
我这样调用没有错嘛。
<img width="721" alt="image" src="https://user-images.githubusercontent.com/15795938/201481399-77c640d6-c31d-40c4-84e6-4df359e86762.png">
然后在这边生成的token和aes对应填上。
$app->getServer(); 然后启用这个服务，我看文档上是这样写的呀。
<img width="298" alt="image" src="https://user-images.githubusercontent.com/15795938/201481437-4790dde0-5692-4028-b8d2-0a3d1d844379.png">
但是点提交后就提示这个错误
openapi回调地址请求不通过
我是想让客服发的消息转发到我报务器上，要配置。老是就不对呢！

**Comment by shuizhuyubuhaochi** [comment]
不知道你用的什么框架    我用的tp出现一样的情况  关掉debug之后就可以了
